src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -4605,7 +4605,7 @@ DECLARE_API(DumpAsync)
                             DacpObjectData objData;
                             if (objData.Request(g_sos, TO_CDADDR(listItemsPtr)) == S_OK && objData.ObjectType == OBJ_ARRAY)
                             {
-                                for (int i = 0; i < objData.dwNumComponents; i++)
+                                for (unsigned int i = 0; i < objData.dwNumComponents; i++)
```

Comment:
```diff
- unsigned int i
+ SIZE_T i
```

too match the type name of `dwNumComponents`.

Reply:
done